Binary files swt-gtk/.git/index and swt-gtk.new/.git/index differ
diff -Nur -x '*.orig' -x '*~' swt-gtk/org/eclipse/swt/accessibility/AccessibleObject.java swt-gtk.new/org/eclipse/swt/accessibility/AccessibleObject.java
--- swt-gtk/org/eclipse/swt/accessibility/AccessibleObject.java	2010-09-13 13:31:45.687685180 -0400
+++ swt-gtk.new/org/eclipse/swt/accessibility/AccessibleObject.java	2010-09-14 18:26:56.111986733 -0400
@@ -1176,7 +1176,10 @@
	}

	static AccessibleObject getAccessibleObject (int /*long*/ atkObject) {
-		return (AccessibleObject)AccessibleObjects.get (new LONG (atkObject));
+		AccessibleObject object = (AccessibleObject)AccessibleObjects.get (new LONG (atkObject));
+		if (object == null) return null;
+		if (object.accessible == null) return null;
+		return object;
	}

	AccessibleObject getChildByHandle (int /*long*/ handle) {
